Subject: Explicitly refer to Python3 in the shebangs
Author: David Steele <steele@debian.org>
Forwarded: no
Last-Update: 2018-06-22

--- a/bin/shoogle
+++ b/bin/shoogle
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 def add_parent_directory_to_sys_path():
     current_directory = os.path.dirname(os.path.realpath(__file__))
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 try:
     import pypandoc
--- a/tests/test_shoogle.py
+++ b/tests/test_shoogle.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """
 test_shoogle
