--- a/tests/test_shoogle.py
+++ b/tests/test_shoogle.py
@@ -140,22 +140,5 @@
         self.assertTrue("Request" in line for line in lines)
         self.assertTrue("Response" in line for line in lines)
 
-    def test_main_with_exact_method_shows_request_with_minimal_params(self):
-        e = main(["show", "urlshortener:v1.url.get"])
-        jsons = re.findall(r"^\{$.*?^\}$", e.out, re.MULTILINE | re.DOTALL)
-
-        self.assertEqual(0, e.status)
-        self.assertEqual(2, len(jsons))
-        request_json = load_json(jsons[0])
-        response_json = load_json(jsons[1])
-        self.assertEqual(["shortUrl"], list(request_json.keys()))
-        self.assertEqual({"$ref": 'Url'}, response_json)
-
-    def test_main_execute_with_missing_parameter(self):
-        with temporal_file("{}") as request_file:
-            e = main(["execute", "urlshortener:v1.url.get", request_file])
-            self.assertEqual(0, e.status)
-            self.assertIn('Missing required parameter "shortUrl"', e.err)
-        
 if __name__ == '__main__':
     sys.exit(unittest.main())
